app-emulation/libvirt/libvirt-7.1.0.ebuild

```diff
@@ -0,0 +1,341 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit meson bash-completion-r1 eutils linux-info python-any-r1 readme.gentoo-r1 systemd
```

Comment:
*eutils.eclass* shouldn't be used for ebuilds anymore. It's functions have been scattered to smaller independant eclasses.